var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -43,13 +43,25 @@ class Elfutils(AutotoolsPackage, SourcewarePackage):
     variant('nls', default=True,
             description='Enable Native Language Support.')
 
+    # libdebuginfod support
+    variant('debuginfod', default=False,
+            description='Enable libdebuginfod support.')
+
     depends_on('bzip2', type='link', when='+bzip2')
     depends_on('xz',    type='link', when='+xz')
     depends_on('zlib',  type='link')
     depends_on('gettext', when='+nls')
     depends_on('m4',    type='build')
 
+    # debuginfod has extra dependencies
+    # NB: Waiting on an elfutils patch before we can use libmicrohttpd@0.9.71
```

Comment:
Can you clarify this comment? If we merge the PR would `elfutils+debuginfod` be functional now that #18226 has been merged?

Reply:
Yes, `elfutils+debuginfod` will be functional, but it cannot build with libmicrohttpd@0.9.71 until an upstream elfutils patch fixes an API change.